.github/actions/behavior_test_binding_python/action.yaml

```diff
@@ -43,6 +43,7 @@ runs:
             working-directory: bindings/python
             run: |
               source venv/bin/activate
+              ruff check .
```

Comment:
We don't need to run ruff check for every behavior test. We can do it in binding_python CI instead, along with build.

Reply:
ok, I find that `.github/workflows/bindings_python.yml` , is digging into it.